packages/components/src/__future__/Select/Select.tsx

```diff
@@ -67,6 +67,7 @@ export type SelectProps<Option extends SelectOption = SelectOption> = {
    * @deprecated: Either define `disabled` in your `Option` (in `items`), or use `disabledKeys`
    */
   disabledValues?: Key[]
+  portalContainer?: Element | DocumentFragment
```

Comment:
Could we instead let the consumer pass in the id of the component they would like to portal to? 

Imagine they have a modal that they're nesting the select into, they could add an `id` to their modal and then the same to their select to create the affect.

Reply:
Probably create an easier API to interact with if they need a portal 🤔 Will give it a crack

Reply:
Updated the component to take an id as recommended. It should offload some of the work folks may have had to do to pass in the element 🕺